.cardboardlint.yml~

```diff
@@ -0,0 +1,11 @@
+linters:
+- import:
+    packages: ['iodata']
+- namespace:
+    filefilter: ['- */__init__.py', '- */test_*.py', '- *setup.py', '- tools/*',
+                 '+ *.py', '+ *.pyx']
+- pylint:
+- pycodestyle:
+    config: .pycodestylerc
+- pydocstyle:
+- whitespace:
```

Comment:
Can you remove this file? I think it is a backup copy that accidentally committed.

Reply:
done